code/game/objects/items/melee/swords.dm

```diff
@@ -0,0 +1,163 @@
+/obj/item/melee/sword
+	name = "broadsword"
+	desc = "A sharp steel forged sword. It's fine edge shines in the light."
+	icon = 'icons/obj/weapons/sword.dmi'
+	icon_state = "broadsword"
+	inhand_icon_state = "broadsword"
+	worn_icon_state = "broadsword"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 20
+	throwforce = 10
+	wound_bonus = 5
+	throw_range = 4
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+	block_chance = 30
+	block_sound = 'sound/weapons/parry.ogg'
+	sharpness = SHARP_EDGED
+	max_integrity = 200
+	armor_type = /datum/armor/item_claymore
+	resistance_flags = FIRE_PROOF
+	embedding = list("embed_chance" = 20, "impact_pain_mult" = 10) //It's a sword, thrown swords can stick into people.
+
+/obj/item/melee/sword/Initialize(mapload)
+	. = ..()
+	AddComponent( \
+		/datum/component/butchering, \
+		speed = 8 SECONDS, \
+		effectiveness = 105, \
+	)
+	AddElement(/datum/element/update_icon_updates_onmob)
+
+/obj/item/melee/sword/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] is falling on [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+	return BRUTELOSS
+
+/obj/item/melee/sword/gold
+	name = "gilded broadsword"
+	desc = "A sharp steel forged sword. It's got a rich guard and pommel. It's fine edge shines in the light."
+	icon_state = "broadsword_gold"
+	inhand_icon_state = "broadsword_gold"
+	worn_icon_state = "broadsword_gold"
+
+/obj/item/melee/sword/rust
+	name = "rusty broadsword"
+	desc = "A sharp steel forged sword. It's edge is rusty and corroded."
+	icon_state = "broadsword_rust"
+	worn_icon_state = "broadsword"
+	force = 15
+	wound_bonus = 0
+	var/broken_icon_state = "broadsword_broken"
```

Comment:
Give these and any other new vars documenting comments (preceded by `///`, above the variable definition) for our autodocumenter explaining what they are for.

Reply:
As an example of how that looks, you would need to do something like this for all of the vars you've included that _are not_ base object vars
```suggestion
	/// Determines the icon state of our sword once it is broken. 
	var/broken_icon_state = "broadsword_broken"
```